web-next/app/codex/skill/page.module.css

```
@@ -0,0 +1,974 @@
+  font-size: 0.7rem;
+  letter-spacing: 0.2em;
+  text-transform: uppercase;
+  color: var(--oai);
+  border: 1px solid rgba(16, 163, 127, 0.3);
+  background: rgba(16, 163, 127, 0.06);
+  border-radius: 2rem;
+  padding: 0.3rem 0.9rem;
+  margin-bottom: 1.4rem;
+}
+.header h1 {
+  font-family: "Syne", sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Syne" (font-family-name-quotes)

(font-family-name-quotes)